tests/wake-format/basic/stdout

```diff
@@ -605,11 +605,7 @@ def tarball Unit =
     require Pass tarball =
         def cmd =
             def gnutar = which "gnutar"
-            def tar =
-                if gnutar ==* "gnutar" then
-                    which "tar"
-                else
-                    gnutar
+            def tar = if gnutar ==* "gnutar" then which "tar" else gnutar
```

Comment:
Yeah, this is about as far as I want to be pushing `if` blocks, but it's still fine for a single line.